src/helpers.js

```diff
@@ -27,6 +29,84 @@ function reduceError (error = {}) {
   return error
 }
 
+/**
+ * Create request options compatible with the console swagger definition
+ *
+ * @param {string} apiKey apiKey to access console api
+ * @param {object} options optional data used for building the request options
+ * @param {object} [options.parameters] parameters to set to the request, specific to each endpoint
+ * @param {object} [options.body] request body for the request
+ * @returns {Array} [{ swaggerParameters }, { requestBody }]
+ */
+function createRequestOptions (apiKey, { parameters = {}, body }) {
+  logger.debug(`params ${JSON.stringify(parameters)}`)
+  logger.debug(`params ${JSON.stringify(body)}`)
+  return [
+    {
+      'x-api-key': apiKey,
+      // This is a fake value, the console swagger spec requires the Authorization parameter
```

Comment:
Thank you for explaining this! Future you will thank you one day too.

Reply:
😊